date_range/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+* Laurent Mignon <laurent.mignon@acsone.eu>
+* Alexis de Lattre <alexis.delattre@akretion.com>
+* Miquel Raïch <miquel.raïch@eficent.com>
```

Comment:
Incorrect e-mail. It should be `miquel.raich@eficent.com`

Reply:
done and rebased